rtCamp/web-stories-wp
# Block/Feature: Uses 'glider-js' for the carousel view.

## Summary

- Updates carousel view to use ['glider-js'](https://nickpiscitelli.github.io/Glider.js/).
- Circles view is displayed as carousel.

## Relevant Technical Choices

- Uses tiny JS library ['glider-js'](https://nickpiscitelli.github.io/Glider.js/) to make carousel feel more consistent on non-AMP and AMP pages.
- Uses ['amp-carousel'](https://amp.dev/documentation/components/amp-carousel/) for AMP pages.

## Testing Instructions

- Insert Web Stories List block with 'circles' or 'carousel' as view type.
- Observe the rendered block.
